OutOfSchool/OutOfSchool.WebApi.Tests/Controllers/WorkshopControllerTests.cs

```diff
@@ -523,6 +534,64 @@ public async Task UpdateWorkshop_WhenIdProviderHasNoRights_ShouldReturn403Object
         Assert.IsNotNull(result);
         Assert.AreEqual(Forbidden, result.StatusCode);
     }
+
+    //[Test]
+    //public async Task UpdateWorkshop_WhenWorkshopDoesntExist_ShouldReturnBadRequestObjectResult()
+    //{
+    //    // Arrange
+    //    workshopUpdateDto.ProviderId = provider.Id;
+    //    providerServiceMoq.Setup(x => x.IsBlocked(provider.Id)).ReturnsAsync(false);
+    //    providerServiceMoq.Setup(x => x.GetByUserId(userId, It.IsAny<bool>()))
+    //        .ReturnsAsync(provider);
+    //    bool? validationResult = null;
+    //    workshopServiceMoq.Setup(x => x.IsAvailableSeatsValidForWorkshop(
+    //        It.IsAny<uint?>(), workshop)).Returns(validationResult);
+
+    //    // Act
+    //    var result = await controller.Update(workshopUpdateDto).ConfigureAwait(false) as ObjectResult;
+
+    //    // Assert
+    //    providerServiceMoq.VerifyAll();
+    //    workshopServiceMoq.Verify(x => x.Update(It.IsAny<WorkshopBaseDto>()), Times.Never);
+    //    Assert.IsNotNull(result);
+    //    Assert.AreEqual(BadRequest, result.StatusCode);
+    //}
+
+    //[Test]
+    //public async Task UpdateWorkshop_WhenDtoAvailableSeatsIsInvalid_ShouldReturnBadRequestObjectResult()
+    //{
+    //    // Arrange
+    //    workshopUpdateDto.ProviderId = provider.Id;
+    //    providerServiceMoq.Setup(x => x.IsBlocked(provider.Id)).ReturnsAsync(false);
+    //    providerServiceMoq.Setup(x => x.GetByUserId(userId, It.IsAny<bool>()))
+    //        .ReturnsAsync(provider);
+    //    bool? validationResult = false;
+    //    workshopServiceMoq.Setup(x => x.IsAvailableSeatsValidForWorkshop(
+    //        It.IsAny<uint?>(), workshopUpdateDto.Id)).ReturnsAsync(validationResult);
+
+    //    // Act
+    //    var result = await controller.Update(workshopUpdateDto).ConfigureAwait(false) as ObjectResult;
+
+    //    // Assert
+    //    providerServiceMoq.VerifyAll();
+    //    workshopServiceMoq.Verify(x => x.Update(It.IsAny<WorkshopBaseDto>()), Times.Never);
+    //    Assert.IsNotNull(result);
+    //    Assert.AreEqual(BadRequest, result.StatusCode);
+    //}
```

Comment:
(Commented code) Please don't leave commented out code.

Reply:
Fixed. Thank you!